docs/contributing.md

```diff
@@ -61,6 +61,15 @@ Not everything is enforced or validated by the schema. A few things to pay atten
 - Nesting of feature identifiers should make sense.
 - Notes use correct grammar and spelling. They should be complete sentences ending with a period.
 
+### Old pull requests
+
+We may close pull requests that have been inactive for 30 or more days.  The procedure is as follows:
+
+1. A peer will review the pull request, and potentially request changes or request more information.
+2. If the pull request author doesn't respond within a week, a peer may ping and ask if the author plans to return to the pull request.
+3. After another week or two, a peer may add a label to mark the pull request as inactive, as well as comment that it's been flagged for closing due to inactivity.
+4. After one final week, a peer may simply close the pull request.  Typically, the pull request would be closed after 30 days of inactivity.
```

Comment:
I'd put the second sentence on a new line, outside the bullets; it is really a comment on the whole procedure, not part of this bullet.